packages/vscode-dependencies-validation/test/commands.spec.ts

```diff
@@ -0,0 +1,111 @@
+import { dirname } from "path";
```

Comment:
I am not a big fan of sinon, but I cant argue with the result that these tests are very clear and understandable 👍

Reply:
Agree